package.json

```diff
@@ -22,38 +22,32 @@
   "dependencies": {
     "archiver": "1.2.0",
     "async": "1.5.2",
-    "body-parser": "1.14.1",
-    "bunyan": "1.5.1",
+    "body-parser": "1.18.2",
     "cors": "2.7.1",
-    "cuid": "1.3.8",
     "diskspace": "1.0.2",
     "express": "4.16.2",
     "express-bunyan-logger": "1.2.0",
     "express-paginate": "0.2.0",
     "fh-agenda": "0.9.0",
     "fh-amqp-js": "0.7.1",
-    "fh-cls-mongoose": "2.1.0",
+    "fh-cls-mongoose": "2.1.1",
     "fh-cluster": "0.3.0",
     "fh-component-metrics": "2.2.1",
     "fh-config": "1.0.3",
-    "fh-forms": "1.16.6",
-    "fh-health": "0.3.1",
+    "fh-forms": "1.16.5",
```

Comment:
I presume this rollback is to patch a regression introduced in 1.16.6?

Reply:
No, this is a mistake in rebase conflict resolution. I'll fix it after next mtg.